crates/ruff_linter/src/rules/flake8_bugbear/rules/reuse_of_groupby_generator.rs

```diff
@@ -102,10 +102,7 @@ impl<'a> GroupNameFinder<'a> {
     }
 }
 
-impl<'a, 'b> Visitor<'b> for GroupNameFinder<'a>
-where
-    'b: 'a,
-{
+impl<'a> Visitor<'a> for GroupNameFinder<'a> {
     fn visit_stmt(&mut self, stmt: &'a Stmt) {
```

Comment:
This is a good example why it's unneded. The lifetime of `stmt` should have been `'b` but it doesn't matter because `'a` and `'b` are equivalent.